examples/tutorials/nb_python/Habitat_Interactive_Tasks.py

```diff
@@ -0,0 +1,2120 @@
+
+                # episode ended
+                if not_done_masks[0].item() == 0:
+                    generate_video(
+                        video_option=self.config.VIDEO_OPTION,
+                        video_dir=self.config.VIDEO_DIR,
+                        images=rgb_frames[0],
+                        episode_id=current_episodes[0].episode_id,
+                        checkpoint_idx=0,
+                        metrics=self._extract_scalars_from_info(infos[0]),
+                        tb_writer=None,
+                    )
+
+                    print("Evaluation Finished.")
+                    print("Success: {}".format(infos[0]["episode_success"]))
+                    print(
+                        "Reward: {}".format(current_episode_reward[0].item())
+                    )
+                    print(
+                        "Distance To Goal: {}".format(
+                            infos[0]["object_to_goal_distance"]
+                        )
+                    )
+
+                    return
+
+                # episode continues
+                elif len(self.config.VIDEO_OPTION) > 0:
+                    frame = observations_to_image(observations[0], infos[0])
+                    rgb_frames[0].append(frame)
+
+
+# %%
+# %load_ext tensorboard
+# %tensorboard --logdir data/tb
+
+# %%
+# !rm -r data/tb
+import random
+
+import numpy as np
+import torch
+
+import habitat
+from habitat import Config, Env, RLEnv, VectorEnv, make_dataset
+from habitat.config import get_config
+from habitat_baselines.config.default import get_config as get_baseline_config
+
+# only run this if in colab environment
+
+baseline_config = get_baseline_config(
+    "habitat_baselines/config/pointnav/ppo_pointnav.yaml"
+)
+baseline_config.defrost()
+
+baseline_config.TASK_CONFIG = config
+baseline_config.TRAINER_NAME = "ddppo"
+baseline_config.ENV_NAME = "RearrangementRLEnv"
+baseline_config.SIMULATOR_GPU_ID = 0
+baseline_config.TORCH_GPU_ID = 0
+baseline_config.VIDEO_OPTION = ["disk"]
+baseline_config.TENSORBOARD_DIR = "data/tb"
+baseline_config.VIDEO_DIR = "data/videos"
+baseline_config.NUM_PROCESSES = 2
+baseline_config.SENSORS = ["RGB_SENSOR", "DEPTH_SENSOR"]
+baseline_config.CHECKPOINT_FOLDER = "data/checkpoints"
+
+if vut.is_notebook():
+    baseline_config.NUM_UPDATES = 400
+else:
+    baseline_config.NUM_UPDATES = 1
+
+baseline_config.LOG_INTERVAL = 10
+baseline_config.CHECKPOINT_INTERVAL = 50
+baseline_config.LOG_FILE = "data/checkpoints/train.log"
+baseline_config.EVAL.SPLIT = "train"
+baseline_config.RL.SUCCESS_REWARD = 2.5
+baseline_config.RL.SUCCESS_MEASURE = "object_to_goal_distance"
+baseline_config.RL.REWARD_MEASURE = "object_to_goal_distance"
+baseline_config.RL.GRIPPED_SUCCESS_REWARD = 2.5
+
+baseline_config.freeze()
+random.seed(baseline_config.TASK_CONFIG.SEED)
+np.random.seed(baseline_config.TASK_CONFIG.SEED)
+torch.manual_seed(baseline_config.TASK_CONFIG.SEED)
+
+trainer = RearrangementTrainer(baseline_config)
```

Comment:
Really surpried this didn't error, still might be best to if __name__ == '__main__': guard here